README.md

```diff
@@ -136,6 +136,10 @@ require("gitlab").setup({
     delete_comment = "dd", -- Delete comment
     reply = "r", -- Reply to comment
     toggle_node = "t", -- Opens or closes the discussion
+    toggle_all_discussions = "T", -- Open or close separately both resolved and unresolved discussions
+    toggle_resolved_discussions = "R", -- Open or close all resolved discussions
+    toggle_unresolved_discussions = "T", -- Open or close all unresolved discussions
```

Comment:
This should be "U" I think

Reply:
@harrisoncramer Sure, thanks for the catch. I was changing it back and forth in three different places and this one I missed :) It's fixed now.